server/atlassian_connect.go

```diff
@@ -37,6 +37,7 @@ func httpACInstalled(p *Plugin, w http.ResponseWriter, r *http.Request) (int, er
 			errors.New("method " + r.Method + " is not allowed, must be POST")
 	}
 
+	defer r.Body.Close()
```

Comment:
I don't think you need to close the Request body on the server. In fact, this may be undesirable.

Reply:
// For server requests, the Request Body is always non-nil
        // but will return EOF immediately when no body is present.
        // The Server will close the request body. The ServeHTTP
        // Handler does not need to.